PyU4V/tests/ci_tests/test_pyu4v_ci_replication.py

```diff
@@ -222,6 +244,26 @@ def test_modify_storage_group_snap_link_by_snap_id(self):
             sg_name, target_sg, snap_name, snap_id, unlink=True)
         self.provisioning.delete_storage_group(target_sg)
 
+    def test_modify_storage_group_snap_link_by_snap_id_relink(self):
+        """Test to cover link snapshot by snap id."""
```

Comment:
do you need to mention relink here?

Reply:
renamed, copy and paste error.  Corrected in latest commit